### PR TITLE
Add current user organizations

### PR DESCRIPTION
**Pull request by medson10**

### CODE REVIEW

File: schema/Connection.re [hunk 1]
```diff
@@ -1,9 +1,56 @@
-type pageInfo = {
-  startCursor: option(string),
-  hasPreviousPage: bool,
-  hasNextPage: bool,
-  endCursor: option(string),
-  total: option(int),
+open Graphql_lwt;
```
Comment (fakenickels) [inline schema/Connection.re]: can you put this under a folder called graphql-helpers? we'll definitely be moving this to other projects

File: schema/Organization.re [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    );
+};
+
+type t = {
+  id: int,
+  name: string,
+  createdAt: string,
+};
+
+let typeResolver =
```
Comment (fakenickels) [inline schema/Organization.re]: I think `resolver` is already good enough

Comment (medson10) [reply schema/Organization.re]: okay

File: schema/Connection.re (hunk 1 above)
Comment (medson10) [reply schema/Connection.re]: okay

File: schema/Organization.re [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+module Project = {
+  type t = {
+    id: int,
+    name: string,
+    webhook: string,
+  };
+
+  let resolver =
+    Schema.(
+      obj("project", ~doc="Project of an organization", ~fields=_ =>
```
Comment (fakenickels) [inline schema/Organization.re]:
```suggestion
      obj("Project", ~doc="Project of an organization", ~fields=_ =>
```
let's enforce type names to be Capitalized

File: schema/Organization.re [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+          ),
+        ]
+      )
+    );
+
+  module Config = {
+    type nodeType = t;
+    type context = Context.t;
+    let nodeResolver = resolver;
+    let nodeName = "project";
```
Comment (fakenickels) [inline schema/Organization.re]:
```suggestion
    let nodeName = "Project";
```

File: schema/Organization.re (hunk 3 above)
Comment (medson10) [reply schema/Organization.re]: I thought in do this, but graphql_lwt uses lower case in some parts so got a bit weird, all mixed, so I thought in keep like this

File: schema/Organization.re [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+        field(
+          "createdAt",
+          ~typ=non_null(string),
+          ~args=Arg.[],
+          ~resolve=(info: Context.t, p: t) =>
+          p.createdAt
+        ),
+        field(
+          "members",
+          ~typ=Member.Connection.connectionResolver,
```
Comment (fakenickels) [inline schema/Organization.re]:
```suggestion
          Connection.connectionResolver((_info, p, first, after, last, before) =>
          None))
```
can we go even further?

Comment (medson10) [reply schema/Organization.re]: pass a callback on this connectionResolver, not getting, why?

Comment (fakenickels) [reply schema/Organization.re]: if not better to change connectionResolver to connectionType

Comment (fakenickels) [reply schema/Organization.re]: it's the resolver right so the user doesn't need to type the arguments again

Comment (fakenickels) [reply schema/Organization.re]:
```
        field(
          "createdAt",
          ~typ=non_null(string),
          ~args=Arg.[],
          ~resolve=(info: Context.t, p: t) =>
          p.createdAt
        ),
        Connection.connectionResolver("member", (_info, p, first, after, last, before) =>
          None)),
```

Comment (medson10) [reply schema/Organization.re]: it's the type resolver

Comment (fakenickels) [reply schema/Organization.re]: the user can pass additional args if wanted to
```
Connection.connectionResolver(~args=[], (_info, p, first, after, last, before) =>
          None))
```

Comment (medson10) [reply schema/Organization.re]: mapping the type return and etc

Comment (fakenickels) [reply schema/Organization.re]: yea the type would be Connection.connectionResolverType actually to keep the naming more accurate

File: schema/Organization.re (hunk 3 above)
Comment (fakenickels) [reply schema/Organization.re]: ah yea guys are doing some weird stuff there

File: schema/Organization.re (hunk 5 above)
Comment (medson10) [reply schema/Organization.re]: yeah that's why I wrote typeResolver before hahah

File: graphql-helpers/Connection.re [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+            ~doc="Node of connection edge",
+            ~typ=non_null(Config.nodeResolver),
+            ~args=Arg.[],
+            ~resolve=(info: Config.context, p: edge) =>
+            p.node
+          ),
+        ]
+      )
+    );
+  let connectionResolver =
```
Comment (fakenickels) [inline graphql-helpers/Connection.re]:
```suggestion
  let connectionType =
```

Comment (fakenickels) [reply graphql-helpers/Connection.re]: `let connectionResolver = (name, cb) => field(name, ~typ=connectionType, ~resolve=cb)`

File: schema/Organization.re (hunk 5 above)
Comment (fakenickels) [reply schema/Organization.re]: https://github.com/Astrocoders/summhub-server/pull/7/files#r260462546

Comment (grsabreu) [reply schema/Organization.re]: d

File: graphql-helpers/Connection.re (hunk 6 above)
Comment (medson10) [reply graphql-helpers/Connection.re]: 🔝

**Review by fakenickels — APPROVED** [review]

File: graphql-helpers/Connection.re [hunk 7]
```diff
@@ -122,16 +122,13 @@ module Create = (Config: Config) => {
...
     Schema.field(
       name,
       ~typ=connectionType,
       ~args=
         Schema.Arg.[
-          arg("first", ~typ=float),
-          arg("after", ~typ=string),
-          arg("last", ~typ=int),
-          arg("before", ~typ=string),
+          ...args
```
Comment (fakenickels) [inline graphql-helpers/Connection.re]: you would still keep the first, last etc etc

**Pull request merged**